src/Utils/Value.php

```diff
@@ -1,12 +1,10 @@
-<?php declare(strict_types=1);
+<?php
+
+declare(strict_types=1);
```

Comment:
Please revert unnecessary formatting changes

Reply:
Formatting is PITA :( 
Every project should have commited their editorconfig / idea settings. Reverting